docs/04_web/03_design-systems/03_preview.md

```diff
@@ -17,10 +17,12 @@ On the left-hand side, is the [Editor](../../../web/design-systems/preview/#edit
 
 Try changing the configurations and observe the reflected changes on the right.
 
-<design-system-editor>
-  <design-system-preview></design-system-preview>
-  <design-system-export slot="export"></design-system-export>
-</design-system-editor>
+<div class="demo-view">
+  <img src="/img/DSC.PNG" alt="Alt Text"/>
+  <div>
+    <a class="overlay-link" href="/dsconfig/" target="_blank">View DSC Demo &#x2197;</a>
```

Comment:
Demo makes it sound like it's a preview of some sort rather than the real thing - I'd just say Launch DSC or smth to that effect